Source/Common/gdcmSwapper.txx

```diff
@@ -127,12 +127,12 @@ namespace gdcm
 
   template <> inline Tag SwapperNoOp::Swap<Tag>(Tag val)
     {
-    return Tag( Swap(val.GetGroup()), Swap(val.GetElement()) );
+    return Tag( Swap((uint32_t)val.GetElementTag()) );
```

Comment:
@malaterre this change I don't know what it does, but it too was to make line 130 the same as line 200. Surely they should be the same?